pom.xml

```diff
@@ -220,7 +221,11 @@
                 <artifactId>postgresql</artifactId>
                 <version>${postgresql.version}</version>
             </dependency>
-
+            <dependency>
```

Comment:
Hi, please move connector dependency to connector's pom.xml. Reference: #2630

Reply:
> Hi, please move connector dependency to connector's pom.xml. Reference: #2630

done